cli/tests/test_op_revert_command.rs

```diff
@@ -0,0 +1,639 @@
+    let work_dir = test_env.work_dir("repo");
+
+    // Double-revert creation of child
+    work_dir.run_jj(["new"]).success();
+    work_dir.run_jj(["op", "revert"]).success();
+    let output = work_dir.run_jj(["op", "revert"]);
+    insta::assert_snapshot!(output, @r"
+    ------- stderr -------
+    Undid operation: 36844b6b7193 (2001-02-03 08:05:09) revert operation dbcb2561b6fee72ea6de79511b6b62f1fff2424f79d16dd30339f94621100f77c86ca7450f7b1ec1bd95d4d56b7a54fe3f3e612353e62cedc682366211b4144e
+    Working copy  (@) now at: rlvkpnrz 43444d88 (empty) (no description set)
+    Parent commit (@-)      : qpvuntsm e8849ae1 (empty) (no description set)
+    [EOF]
+    ");
+
+    // Double-revert creation of sibling
+    work_dir.run_jj(["new", "@-"]).success();
+    work_dir.run_jj(["op", "revert"]).success();
+    let output = work_dir.run_jj(["op", "revert"]);
+    insta::assert_snapshot!(output, @r"
+    ------- stderr -------
+    Undid operation: 205f00eaed13 (2001-02-03 08:05:12) revert operation 791c1da840ca87bea33edfef8f75fe54d7f83b6784f6126d43e0ad212deefd638a3286a6466037aa28de8ed662c0d4219540ebf7118d9946d96781a5bf094100
+    Working copy  (@) now at: mzvwutvl 8afc18ff (empty) (no description set)
+    Parent commit (@-)      : qpvuntsm e8849ae1 (empty) (no description set)
+    [EOF]
+    ");
+}
+
+#[test]
+fn test_revert_latest_revert_explicitly() {
+    let test_env = TestEnvironment::default();
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
+    let work_dir = test_env.work_dir("repo");
+
+    work_dir.run_jj(["new"]).success();
+    work_dir.run_jj(["op", "revert"]).success();
+    let output = work_dir
+        .run_jj(["op", "log", "--no-graph", "-T=id.short()", "-n=1"])
+        .success();
+    let op_id_hex = output.stdout.raw();
+    let output = work_dir.run_jj(["op", "revert", op_id_hex]);
+    insta::assert_snapshot!(output, @r"
+    ------- stderr -------
+    Undid operation: 36844b6b7193 (2001-02-03 08:05:09) revert operation dbcb2561b6fee72ea6de79511b6b62f1fff2424f79d16dd30339f94621100f77c86ca7450f7b1ec1bd95d4d56b7a54fe3f3e612353e62cedc682366211b4144e
+    Working copy  (@) now at: rlvkpnrz 43444d88 (empty) (no description set)
+    Parent commit (@-)      : qpvuntsm e8849ae1 (empty) (no description set)
+    [EOF]
+    ");
+
+    work_dir.run_jj(["new"]).success();
+    work_dir.run_jj(["op", "revert"]).success();
+    let output = work_dir.run_jj(["op", "revert", "@"]);
+    insta::assert_snapshot!(output, @r"
+    ------- stderr -------
+    Undid operation: ca6c7df545ed (2001-02-03 08:05:13) revert operation f711a03983b4103310aa6c16b943cc59decf1ddc0dcb2f38fddeb2c1ebe358ed8753938d311199d3e13e081a28749831844dea6e538a407d1a72a6664f2397d2
+    Working copy  (@) now at: royxmykx ba0e5dca (empty) (no description set)
+    Parent commit (@-)      : rlvkpnrz 43444d88 (empty) (no description set)
+    [EOF]
+    ");
+}
+
+#[test]
+fn test_revert_an_older_revert() {
```

Comment:
This and the other two tests were added by ca7a6fcbce17b05919ad440044845efca6d63d6d, so they would belong to the double undo warning tests?

Reply:
I can see that `test_undo_bookmark_deletion` doesn't make sense in this file. But do the other two tests maybe have value for `jj op revert`? As noted above, the behavior these are testing will soon become deprecated for `jj undo`.